### PR TITLE
Fix the Uncaught TypeError

### PR DESCRIPTION
**Pull request by HunseopJeong**
[Issue] N/A
[Problem] Uncaught Error Occurred
[Solution] Remove undefined _unbindEvents

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

**Comment by HunseopJeong** [comment]
image

File: src/js/core/widget/core/Card.js [hunk 1]
```diff
@@ -179,7 +179,6 @@
 			 * @protected
 			 */
 			prototype._destroy = function () {
```
Comment (TomaszLukawskiSam) [inline src/js/core/widget/core/Card.js]: I think you can remove whole method if this method does nothing.

Comment (HunseopJeong) [reply src/js/core/widget/core/Card.js]: Done

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Pull request merged**